neural_lam/models/ar_model.py

```diff
@@ -96,6 +95,7 @@ def __init__(
             # Store constant per-variable std.-dev. weighting
             # NOTE that this is the inverse of the multiplicative weighting
             # in wMSE/wMAE
+            # TODO: Do we need param_weights for this?
```

Comment:
This seems unrelated and just an artifact from merging/rebasing.

Reply:
Fixed with https://github.com/mllam/neural-lam/pull/77/commits/ae69f3f32d0add5a47db1edefaf8de36129ff58d